Backend/NoCode.FlowerShop.Api/Controllers/FlowersController.cs

```diff
@@ -25,23 +24,11 @@ public FlowersController(ISender mediator, IMapper mapper)
     [AuthorizeRoles(UserRole.Administrator)]
     public async Task<IActionResult> Create(CreateFlowerRequest request)
     {
-        var command = _mapper.Map<CreateFlowerCommand>(request);
-        var result = await _mediator.Send(command);
-        
-        return result.Match(
-            result => Ok(_mapper.Map<CreateFlowerResponse>(result)),
-            errors => Problem(errors));
-    }
+        var query = _mapper.Map<CreateFlowerCommand>(request);
+        var result = await _mediator.Send(query);
 
-    [HttpDelete("{id:guid}")]
-    [AuthorizeRoles(UserRole.Administrator)]
-    public async Task<IActionResult> Delete(Guid id)
-    {
-        var command = new DeleteFlowerCommand(id);
-        var result = await _mediator.Send(command);
-        
         return result.Match(
-            result => Ok(_mapper.Map<DeleteFlowerResponse>(result)),
+            result => Ok(_mapper.Map<CreateFlowerResponse>(result)),
```

Comment:
Please make sure you will be merging the right version of main, when the issue with repo is resolved